src/views/Play/Play.tsx

```diff
@@ -545,9 +551,16 @@ export class Play extends React.Component<{}, PlayState> {
                                 {this.cellBreaks(7)}
```

Comment:
It looks like these cell breaks are pushing everything to the left

<img width="1326" alt="Screen Shot 2022-01-13 at 8 43 16 PM" src="https://user-images.githubusercontent.com/25233703/149452908-a838f490-0821-4f98-bc7e-8f1adb743285.png">